flink-runtime/src/test/java/org/apache/flink/runtime/operators/util/OutputEmitterTest.java

```diff
@@ -211,26 +214,35 @@ public void testWrongKeyClass() throws Exception {
 
 		try {
 			delegate.setInstance(record);
-			selector.selectChannels(delegate, 100);
+			selector.selectChannels(delegate);
 		} catch (DeserializationException re) {
 			return;
 		}
 		Assert.fail("Expected a NullKeyFieldException.");
 	}
 
+	private ChannelSelector createChannelSelector(
+		ShipStrategyType shipStrategyType,
+		TypeComparator comparator,
+		int numChannels) {
```

Comment:
same comment as before: formatting (method parameters and the method code are melding together)

Reply:
yes